### PR TITLE
[MIG] account_banking_mandate_sale: 16.0

### PR DESCRIPTION
**Pull request by ferran-S73**

### CODE REVIEW

**Comment by CasVissers-360ERP** [comment]
@ferran-S73 I get two warnings:
The model odoo.addons.account_banking_mandate.models.account_banking_mandate is not overriding the create method in batch 
The model odoo.addons.account_banking_mandate.models.account_move is not overriding the create method in batch

**Comment by ferran-S73** [comment]
> @ferran-S73 I get two warnings: The model odoo.addons.account_banking_mandate.models.account_banking_mandate is not overriding the create method in batch The model odoo.addons.account_banking_mandate.models.account_move is not overriding the create method in batch

Hi @CasVissers-360ERP that seems weird to me since this module isn't inheriting any of those create methods, where are you getting the warnings?

File: account_banking_mandate_sale/oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+sale-workflow
```
Comment (astirpe) [inline account_banking_mandate_sale/oca_dependencies.txt]: This file `oca_dependencies.txt` should be not placed inside the module. Use the one already existing in the repo https://github.com/OCA/bank-payment/tree/16.0

Comment (ferran-S73) [reply account_banking_mandate_sale/oca_dependencies.txt]: You're right, I don't know who put it there it comes from a previous version. I'll remove it

**Review by astirpe — APPROVED** [review]

**Comment by alexis-via** [comment]
I improved this migration in this other PR https://github.com/OCA/bank-payment/pull/1100

**Pull request closed without merging**